xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/InplaceEditing.xml

```diff
@@ -768,6 +774,7 @@ define('editInPlace', [
 
   var initTranslateButton = function(xwikiDocument) {
     // Initialize the translate button only if it's visible.
+    // FIXME: this should probably use the translateButtonSelector
     const translateButton = $('#tmTranslate [data-toggle="popover"]').filter(':visible');
```

Comment:
```suggestion
    const translateButton = $(config.translateButtonSelector).filter('[data-toggle="popover"]').filter(':visible');
```

Reply:
Note that the current selector is not `#tmTranslate` but `#tmTranslate > a`, that's why I hesitated modifying the selector there. You're sure it would work properly with this?

Reply:
It should work fine because ``$('#tmTranslate > a')`` finds the anchors (it should be just one) and then ``.filter('[data-toggle="popover"]').filter(':visible')`` filters this list of anchors (with just one item) based on whether the anchors have the ``data-toggle`` attribute and are visible. Basically it keeps in the list only the ``'#tmTranslate > a'`` anchors that are visible and have the ``data-toggle="popover"`` attribute, which is what we want.

Reply:
Thanks for the confirmation